Extension/.vscode/launch.json

```diff
@@ -10,7 +10,7 @@
 			"args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
 			"stopOnEntry": false,
 			"sourceMaps": true,
-			"outDir": "${workspaceRoot}/out/src",
+			"outFiles": ["${workspaceRoot}/out/src"],
```

Comment:
Reading the helptext on outFiles, looks like its a glob pattern to specify js files. do we want to set this to `
"${workspaceRoot}/out/src/**/*.js"` ?